docs-source/content/userguide/managing-domains/domain-lifecycle/introspection.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Domain introspection"
 date: 2020-07-07T08:14:51-05:00
-draft: true
+draft: false
```

Comment:
@rosemarymarano, I'll likely move this change to its own PR so that we can get it in while we review (or debate) the rest of the content here.